src/plugins/resolver/resolver.c

```diff
@@ -73,6 +73,9 @@ static void resolverInit (resolverHandle * p, const char * path)
 	p->tempfile = 0;
 
 	p->path = path;
+
+	p->uid = getuid ();
+	p->gid = getgid ();
```

Comment:
I think thats wrong, we are not interested in chown to our own user (thats the case anyway), but to do a chown in the case of root to the uid/gid that was present in the file before.